TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/workflow/commands/status.py

```diff
@@ -0,0 +1,376 @@
+
+
+def _get_step_rich_label(node: dict) -> str:
+    """Get rich-formatted label for a workflow step node.
+
+    Args:
+        node: WorkflowNode dictionary
+
+    Returns:
+        Rich-formatted string with color and styling
+    """
+    step_name = node['display_name']
+    step_phase = node['phase']
+    step_type = node['type']
+
+    # Color based on phase
+    if step_phase == 'Succeeded':
+        color = "green"
+        symbol = "✓"
+    elif step_phase == 'Running':
+        color = "yellow"
+        symbol = "⟳" if step_type == 'Suspend' else "▶"
+    elif step_phase in ('Failed', 'Error'):
+        color = "red"
+        symbol = "✗"
+    elif step_phase == 'Pending':
+        color = "cyan"
+        symbol = "○"
+    elif step_phase == 'Skipped':
+        color = "dim"
+        symbol = "~"
+    else:
+        color = "white"
+        symbol = "?"
+
+    # Special handling for Suspend steps
+    if step_type == 'Suspend':
+        if step_phase == 'Running':
+            return f"[{color}]{symbol} {step_name} - WAITING FOR APPROVAL[/{color}]"
+        elif step_phase == 'Succeeded':
+            return f"[{color}]{symbol} {step_name} (Approved)[/{color}]"
+        else:
+            return f"[{color}]{symbol} {step_name} ({step_phase})[/{color}]"
+    # Special handling for Skipped steps with approval-related names
+    elif step_phase == 'Skipped' and 'approval' in step_name.lower():
+        return f"[{color}]{symbol} {step_name} (Not Required)[/{color}]"
+    else:
+        return f"[{color}]{symbol} {step_name} ({step_phase})[/{color}]"
+
+
+def _display_workflow_tree(step_tree: list):
+    """Display workflow steps in tree format using Rich.
+
+    Args:
+        step_tree: List of WorkflowNode dictionaries with hierarchy
+    """
+    if not step_tree:
+        return
+
+    console = Console()
+    tree = Tree("📋 [bold]Workflow Steps[/bold]")
+
+    # Group nodes by depth to build the tree structure
+    nodes_by_depth = {}
+    for node in step_tree:
+        depth = node['depth']
+        if depth not in nodes_by_depth:
+            nodes_by_depth[depth] = []
+        nodes_by_depth[depth].append(node)
+
+    # Build tree level by level
+    node_to_tree = {}
+
+    for depth in sorted(nodes_by_depth.keys()):
+        for node in nodes_by_depth[depth]:
+            label = _get_step_rich_label(node)
+
+            if depth == 0:
+                # Root level nodes
+                node_to_tree[node['id']] = tree.add(label)
+            else:
+                # Find parent and add as child
+                parent_id = node.get('parent')
+                if parent_id and parent_id in node_to_tree:
+                    node_to_tree[node['id']] = node_to_tree[parent_id].add(label)
+                else:
+                    # Fallback: add to root if parent not found
+                    node_to_tree[node['id']] = tree.add(label)
+
+    click.echo("")
+    console.print(tree)
```

Comment:
I like the tree output.  I'd also like to see json output so that I could do some jpath (jq) queries to get names that I could then check for or do other scripting with.